templates/module/src/entity-content-route-provider.php.twig

```diff
@@ -161,7 +161,7 @@ class {{ entity_class }}HtmlRouteProvider extends AdminHtmlRouteProvider {% endb
       $route = new Route("/admin/structure/{$entity_type->id()}/settings");
       $route
         ->setDefaults([
-          '_form' => 'Drupal\{{ module }}\Controller\{{ entity_class }}SettingsForm',
+          '_form' => 'Drupal\{{ module }}\Form\{{ entity_class }}SettingsForm',
```

Comment:
Have you created an entity without a bundle type? In this case, you are right, this command does not generate the Controller file but in the code still mentions it (specifically in RouteProvider). This not mean that it will try to use it because there are not a bundle type (see the line 91). When you create an entity with bundle type, the Controller will be created.

So I think that the Controller really should not be mentioned in the code if does not really exist. I'm going to make some changes.